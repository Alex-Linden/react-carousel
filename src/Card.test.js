import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

it("renders without crashing", function () {
  render(<Card
    caption="smoke and mirrors"
    src="test.com"
    currNum={1}
    totalNum={4} />);
});

it("matches snapshot", function () {
  const { container, debug } = render(<Card
    caption="smoke and mirrors"
    src="test.com"
    currNum={1}
    totalNum={4} />);

  expect(container).toMatchSnapshot();
});


// const img = container.querySelector("img");
//   debug(img);