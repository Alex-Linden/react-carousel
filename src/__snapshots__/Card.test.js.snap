// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      smoke and mirrors
    </h4>
    <img
      alt="smoke and mirrors"
      class="Card-image"
      src="test.com"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      4
      .
    </small>
  </div>
</div>
`;
